### PR TITLE
aio: implement remark renderer

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

File: aio/content/examples/cli-quickstart/src/styles.css [hunk 1]
```diff
@@ -0,0 +1 @@
+undefined
```
Comment (Foxandxss) [inline aio/content/examples/cli-quickstart/src/styles.css]: this should not be undefined. Be sure you have this file.

Comment (petebacondarwin) [inline aio/content/examples/cli-quickstart/src/styles.css]: @Foxandxss - so I pulled from angular.io master. Are you saying that I need to generate the boilerplate before running migration?

Comment (Foxandxss) [inline aio/content/examples/cli-quickstart/src/styles.css]: No, that file should exist already.

Comment (petebacondarwin) [inline aio/content/examples/cli-quickstart/src/styles.css]: hmm then this is a bug in the migrator...

**Review by IgorMinar — APPROVED** [review]

**Pull request merged**